.github/workflows/push-csv-to-repo.yml

```diff
@@ -3,17 +3,26 @@ name: Push CSV to Repo
 
 on:
   workflow_dispatch:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+  schedule:
+    - cron: '0 12 * * *' # runs at 12:00 UTC everyday
 
 jobs:
   build:
     runs-on: ubuntu-latest
     container: pandoc/latex
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.7.16' # install the python version needed
```

Comment:
What is our current version of Python? Got `2.7.16` from my venv but don't think it's right

Reply:
3.8.3 see https://github.com/serotracker/iit-backend/blob/master/.github/workflows/deploy-eb.yml#L31